### PR TITLE
fix(docstring): fix docstring for XGB predictor. 

### PR DESCRIPTION
**Pull request by NikeNano**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/apis/serving/v1beta1/predictor_xgboost.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// SKLearnSpec defines arguments for configuring SKLearn model serving.
+// XGBoostSpec defines arguments for configuring XGBoost model serving.
```
Comment (yuzisun) [inline pkg/apis/serving/v1beta1/predictor_xgboost.go:29]: thanks for spotting this! can you help fix [here](https://github.com/kubeflow/kfserving/blob/7452690e7188755c2b45d785ff5371b4b1451690/pkg/apis/serving/v1beta1/swagger.json#L3778) too?
you can run `hack/update-openapigen.sh` to update that.

Comment (NikeNano) [reply pkg/apis/serving/v1beta1/predictor_xgboost.go:29]: Sure, will fix it.

Comment (NikeNano) [reply pkg/apis/serving/v1beta1/predictor_xgboost.go:29]: Fixed @yuzisun

**Comment by yuzisun** [comment]
thanks @NikeNano for the fix!

/lgtm
/approve

**Pull request merged**